lib/loaders/gltf.ts

```diff
@@ -1,12 +1,20 @@
 import type { CoordinateTransformConfig, STLMesh, OBJMesh } from "../types"
 import { parseGLB } from "./glb"
+import * as path from "path"
 
 async function fetchAsArrayBuffer(url: string): Promise<ArrayBuffer> {
-  const response = await fetch(url)
-  if (!response.ok) {
-    throw new Error(`Failed to fetch ${url}: ${response.statusText}`)
+  if (
+    url.startsWith("http:") ||
+    url.startsWith("https:") ||
+    url.startsWith("data:")
+  ) {
+    const response = await fetch(url)
+    if (!response.ok) {
+      throw new Error(`Failed to fetch ${url}: ${response.statusText}`)
+    }
+    return response.arrayBuffer()
   }
-  return response.arrayBuffer()
+  return Bun.file(url).arrayBuffer()
```

Comment:
we can't depend on bun internally (only in test files) otherwise we break node compatibility, also `fetch` should work on `file://` urls